includes/blocks/example-block/register.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Example-block
+ * Custom title block -- feel free to delete
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { registerBlockType } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
```

Comment:
@ryanwelcher would it be cleaner to maybe organise the folder so that we could export out something like: `import { edit, save, json } from './file'`